File: Euler 0001 - Multiples of 3 and 5/euler_0001.scala
// https://projecteuler.net/problem=1
// created 12.30.2017 by CB Fay

object solution {
    def main(args : Array[String]) {
        var multiples : Set[Int] = Set()
        
        for (i <- 3 until 1000 if i % 3 == 0) {
            multiples += i
        }
        
        for (i <- 5 until 1000 if i % 5 == 0) {
            multiples += i
        }
        
        println(multiples.sum)
    }
}

File: Euler 0001 - Multiples of 3 and 5/euler_0001.py
# https://projecteuler.net/problem=1
# created 12.30.2017 by CB Fay

def answer():
    multiples = set()
    for i in range(3,1000,3):
        multiples.add(i) 
    for i in range(5,1000,5):
        multiples.add(i)
    print(sum(multiples))


answer()
